app/apps/cases/models.py

```diff
@@ -25,43 +40,46 @@ class Meta:
     )
     is_legacy_bwv = models.BooleanField(default=False)
     camunda_id = models.CharField(max_length=255, null=True, blank=True)
+    case_team = models.ForeignKey(to=CaseTeam, null=True, on_delete=models.PROTECT)
+    case_reason = models.ForeignKey(to=CaseReason, null=True, on_delete=models.PROTECT)
+    text = models.TextField(blank=True, null=True)
```

Comment:
Variable name moet omschrijvend zijn

Reply:
Done. In de frontend hadden ze dit met 'text' gemocked, maar nu heb ik het hernoemd naar 'description' wat duidelijker is. Dus ff afstemmen met de frontenders als dit merged is.